packages/kit-bg/src/states/jotai/atoms/settings.ts

```
@@ -1,4 +1,4 @@
-import type { ILocaleSymbol } from '@onekeyhq/shared/src/locale';
+import { ETranslations, ILocaleSymbol } from '@onekeyhq/shared/src/locale';
```

ESLint

[error] Import "ILocaleSymbol" is only used as types.

(@typescript-eslint/consistent-type-imports)